src/Parser/Monk/Mistweaver/Modules/Spells/AzeriteTraits/FontOfLife.js

```diff
@@ -38,12 +47,22 @@ class FontOfLife extends Analyzer {
 
       this.baseHeal = (intRating * ESSENCE_FONT_SPELLPOWER_COEFFICIENT) * mwAura * (1 + versPerc);
 
-      if (event.hitType === 2) {
+      if (event.hitType === HIT_TYPES.CRIT) {
         this.baseHeal = this.baseHeal * 2;
       }
       this.healing += (healAmount - this.baseHeal);
     }
   }
+
+  subStatistic() {
+    return (
+      <StatisticListBoxItem
+        title={<SpellLink id={SPELLS.FONT_OF_LIFE.id} />}
+        value={<dfn data-tip={`Added a total of ${formatNumber(this.healing)} to your Essence Font Bolts.`}>{formatPercentage(this.healing / this.getAbility(SPELLS.ESSENCE_FONT_BUFF.id).healingEffective)} % of Essence Font Healing</dfn>}
```

Comment:
Sorry to intrude, there's also a string `valueTooltip` property you can use instead of this :)

Reply:
Good catch 👍

Reply:
Heh - No problem - I will make sure to modify it on my new PR to use the full functionality.
Sorry about that.